src/controllers/dlgprefcontroller.cpp

```diff
@@ -462,13 +462,24 @@ void DlgPrefController::savePreset() {
         return;
     }
 
-    if (m_pPreset->filePath().startsWith(resourcePresetsPath(m_pConfig))) {
```

Comment:
Oh good, I was just going to ask you to append "(edited)" to edited presets from the user folder too.